src/kvilib/net/KviSSL.h

```diff
@@ -92,7 +92,7 @@ class KVILIB_API KviSSLCertificate
 	int publicKeyBits() { return m_iPubKeyBits; };
 	const char * publicKeyType() { return m_szPubKeyType.ptr(); };
 
-	int serialNumber() { return m_iSerialNumber; };
+	const char * serialNumber() { return m_szSerialNumber.len() ? m_szSerialNumber.ptr() : nullptr; };
```

Comment:
Does `nullptr` need to be returned instead of a single byte string containing only `'\0'`?  I don't see this being an issue and I don't see any of the other returns making this distinction.